docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -263,37 +263,29 @@ pipeline {
 
 ## CircleCI
 
-```yaml
```

Comment:
Looks like an accidental yaml removal in the line?

Reply:
I removed the yaml syntax as I was having issues with a few lines; the yaml syntax wasn't interpreting them as comments and kept highlighting them red, since they were more like bash statements than comments, so I left out the yaml syntax. :/

Reply:
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/50721067/189456278-3007b55e-6f0e-481a-b650-5f7cbd452e55.png">
Those echo statements towards the bottom shouldn't be red, they should be commented out; would you happen to know how to get those to comment?

Reply:
Looks like the renderer doesn't like pipes (|). Let's go ahead and merge it without the yaml. Thanks!